<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyClassClass</class>
 <widget class="QDialog" name="MyClassClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyClass</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="currentButton">
       <property name="text">
        <string>打开初始图</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="doButton">
       <property name="text">
        <string>处理道路</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="newButton">
       <property name="text">
        <string>打开新图</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="myclass.qrc"/>
 </resources>
 <connections/>
</ui>
